hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/artifacts/smtable/SoftwareModuleTableHeader.java

```diff
@@ -42,51 +46,50 @@ void onEvent(final UploadArtifactUIEvent event) {
 
     @Override
     protected String onLoadSearchBoxValue() {
-        return artifactUploadState.getSoftwareModuleFilters().getSearchText().orElse(null);
+        return getArtifactUploadState().getSoftwareModuleFilters().getSearchText().orElse(null);
```

Comment:
Ok then why do you need line 37 and the member variable `ArtifactUploadState artifactUploadState`?